lib/.gitignore

```diff
@@ -0,0 +1,18 @@
+/Makefile.am
```

Comment:
We would remove this and add the dummy lib/Makefile.am, if we decide to go that route.  I'm open to better suggestions (to make it build on Linux w/o the gnulib import)